### PR TITLE
ci: try sccache

### PR DESCRIPTION
**Pull request by xxchan**
I hereby agree to the terms of the [RisingWave Labs, Inc. Contributor License Agreement](https://gist.github.com/TennyZhuang/f00be7f16996ea48effb049aa7be4d66#file-rw_cla).

### CODE REVIEW

**Comment by xxchan** [comment]
<details><summary>First run  24min55s </summary>
image
</details>

<details><summary>First run  22min49s </summary>
image
</details>

File: ci/Dockerfile [hunk 1]
```diff
@@ -30,9 +30,12 @@ ENV PATH /root/.cargo/bin/:$PATH
...
 # install build tools
-RUN cargo install cargo-llvm-cov cargo-nextest cargo-udeps cargo-hakari cargo-sort cargo-make cargo-cache \
+RUN cargo install cargo-llvm-cov cargo-nextest cargo-udeps cargo-hakari cargo-sort cargo-make cargo-cache sccache \
     && cargo install --git https://github.com/risinglightdb/sqllogictest-rs --rev dbadddb --bin sqllogictest --locked \
     && cargo cache -a \
     && rm -rf "/root/.cargo/registry/index" \
     && rm -rf "/root/.cargo/registry/cache" \
     && rm -rf "/root/.cargo/git/db"
+
+ENV RUSTC_WRAPPER=sccache
```
Comment (BugenZhao) [inline ci/Dockerfile:40]: Hard to believe it's so easy to use. 🥵

**Review by BugenZhao — APPROVED** [review]

File: ci/Dockerfile (hunk 1 above)
Comment (xxchan) [reply ci/Dockerfile:40]: 🥵 The hard part for me is to learn to how to configure AWS

**Review by jon-chuang — APPROVED** [review]
LGTM!

File: ci/Dockerfile (hunk 1 above)
Comment (jon-chuang) [reply ci/Dockerfile:40]: Built by Mozilla after all 😍

**Pull request merged**